mbuild/tests/test_compound.py

```diff
@@ -68,6 +68,17 @@ def test_save_forcefield_with_file(self, methane):
                          forcefield_files=get_fn('methane_oplssaa.xml'),
                          overwrite=True)
 
+    @pytest.mark.skipif(not has_foyer, reason="Foyer is not installed")
+    def test_save_forcefield_with_file_foyerkwargs(self, methane):
+        exts = ['.gsd', '.hoomdxml', '.lammps', '.lmp', '.top', '.gro',
+                '.mol2', '.pdb', '.xyz']
+        foyerkwargs = {'assert_dihedral_params': False}
+        for ext in exts:
+            methane.save('lythem' + ext,
+                         forcefield_files=get_fn('methane_oplssaa.xml'),
+                         overwrite=True, foyerkwargs=foyerkwargs)
```

Comment:
How do we make sure the kwargs are getting passed? Is the idea that the typing fails if it tries `assert_dihedral_params=True`